package.json

```diff
@@ -17505,6 +17505,7 @@
 		"https-proxy-agent": "5.0.1",
 		"iconv-lite": "0.6.3",
 		"lit": "3.1.3",
+		"marked": "12.0.2",
```

Comment:
`yarn.lock` is not updated

Reply:
That was on purpose to avoid conflicts. I think that is something we should do generally, because it just causes issues when merging. Though it is needed for the ultimate merge so it adds another merge step.

Reply:
Isn't it a good thing that it causes conflicts (i.e. wouldn't we want to catch/fix these conflicts ahead of time, rather than on merge?) or am I missing something?